.github/CHANGELOG.md

```diff
@@ -1,3 +1,79 @@
+# Release 0.4.0dev0
+
+### New features since last release
+
+* `pennylane.about()` function added, providing details
+  on current PennyLane version, installed plugins, Python,
+  platform, and NumPy versions [#186](https://github.com/XanaduAI/pennylane/pull/186)
+
+* Removed the logic that allowed `wires` to be passed as a positional
+  argument in quantum operations. This allows us to raise more useful
+  error messages for the user if incorrect syntax is used.
+  [#188](https://github.com/XanaduAI/pennylane/pull/188)
+
+* Adds support for multi-qubit expectation values of the `pennylane.Hermitian()`
+  observable [#192](https://github.com/XanaduAI/pennylane/pull/192)
+
+* Adds support for multi-qubit expectation values in `default.qubit`.
+  [#202](https://github.com/XanaduAI/pennylane/pull/202)
+
+* Organise templates into submodules [#195](https://github.com/XanaduAI/pennylane/pull/195).
+  This included the following improvements:
+
+  - Distinguish embedding templates, layer templates and parameter templates
+
+  - Add Random Circuit, in which rotations and 2-qubit gates are randomly
```

Comment:
Should we use the actual (code) template name?

Reply:
```suggestion
  - Added a random circuit template (`RandomLayers()`), in which rotations and 2-qubit gates are randomly
```

Reply:
done